content/src/service/synchronization/SynchronizationManager.ts

```diff
@@ -95,8 +100,10 @@ export class ClusterSynchronizationManager implements SynchronizationManager {
             this.synchronizationState = SynchronizationState.FAILED_TO_SYNC;
             ClusterSynchronizationManager.LOGGER.warn(`Failed to sync with servers. Reason:\n${error}`)
         } finally {
-            // Set the timeout again
-            this.syncWithNodesTimeout = setTimeout(() => this.syncWithServers(), this.timeBetweenSyncs)
+            if (!this.stopping) {
```

Comment:
Maybe you can also add this checking at the beginning of syncWithServers.

Reply:
It shouldn't be necessary, since we are clearing the timeout.